zerochain/native/src/lib.rs

```diff
@@ -1,14 +1,25 @@
 #[macro_use]
 extern crate neon;
-extern crate num_cpus;
-
+extern crate dirs;
```

Comment:
Can remove this.
You don't have to import libs by `extern crate` in `edition 2018` syntax.